src/js/dp/release-calendar.js

```diff
@@ -30,3 +36,118 @@ document.addEventListener("DOMContentLoaded", function () {
     releaseTypeAutoSubmit(".release-calendar__filters .filters__release-type");
   }
 });
+
+const sortSelector = document.querySelector(".ons-input--select");
+if (sortSelector) {
+  sortSelector.addEventListener("change", async (e) => {
+    gtmDataLayerPush({
+      event: "SortBy",
+      "sort-by": e.target.value,
+    });
+  });
+}
+
+if (releaseCalendarContainer) {
+  [
+    ...releaseCalendarContainer.querySelectorAll(
+      ".ons-radio__input[type=radio]:not(input:disabled)"
+    ),
+  ].map((topFilter) => {
+    topFilter.addEventListener("change", async (e) => {
+      gtmDataLayerPush({
+        event: "Filter",
+        "filter-by": e.target.name,
+        selected: e.target.value.replace("type-", ""),
+      });
+    });
+  });
+
+  document.querySelector('[aria-label="Keywords"]').onsubmit = async (e) => {
+    const formData = new FormData(e.target);
+    const formProps = Object.fromEntries(formData);
+    gtmDataLayerPush({
+      event: "Filter",
+      "filter-by": "search",
+      "search-term": formProps.keywords,
+    });
+  };
+
+  document.querySelector('[aria-label="Released date"]').onsubmit = async (
+    e
+  ) => {
+    const formData = new FormData(e.target);
+    const formProps = Object.fromEntries(formData);
+    const startDate = `${formProps["after-year"]}/${formProps["after-month"]}/${formProps["after-day"]}`;
+    const endDate = `${formProps["before-year"]}/${formProps["before-month"]}/${formProps["before-day"]}`;
+
+    const numberOfDays = days(
+      new Date(
+        formProps["after-year"],
+        formProps["after-month"],
+        formProps["after-day"]
+      ),
+      new Date(
+        formProps["before-year"],
+        formProps["before-month"],
+        formProps["before-day"]
+      )
+    );
+
+    gtmDataLayerPush({
+      event: "Filter",
+      "filter-by": "date-range",
+      "start-date": startDate,
+      "end-date": endDate,
+      "number-of-days": numberOfDays,
+    });
+  };
+}
+
+if (releasePageContainer) {
+  const releaseStatus = releasePageContainer.getAttribute(
```

Comment:
More specifically with `data-` attributes, iirc this should work instead of the more general `getAtttribute()`:

`const releaseStatus = releasePageContainer.dataset.gtmReleaseStatus`